motr/io_nw_xfer.c

```diff
@@ -402,6 +410,11 @@ void target_ioreq_fini(struct target_ioreq *ti)
 		m0_fop_put_lock(&ti->ti_cc_fop.crf_fop);
 	}
 
+	if ( opcode == M0_OC_WRITE ) {
```

Comment:
@yeshpal-jain-seagate : We have a comment on line 395 which says that ti_attrbuf should not be freed for write path. But on this line we are freeing ti_attrbuf. Should we remove the comment ?